src/components/FileList.tsx

```diff
@@ -340,6 +346,48 @@ export class FileList extends React.Component<IFileListProps, IFileListState> {
     );
   }
 
+  /**
+   * Render an unmerged file
+   *
+   * Note: This is actually a React.FunctionComponent but defined as
+   * a private method as it needs access to FileList properties.
+   *
+   * @param rowProps Row properties
+   */
+  private _renderUnmergedRow = (
+    rowProps: ListChildComponentProps
+  ): JSX.Element => {
+    const { data, index, style } = rowProps;
+    const file = data[index] as Git.IStatusFile;
+    const diffButton = this._createDiffButton(file);
+    return (
+      <FileItem
+        trans={this.props.trans}
+        actions={!file.is_binary && diffButton}
+        contextMenu={this.openContextMenu}
+        file={file}
+        model={this.props.model}
+        selected={this._isSelectedFile(file)}
+        selectFile={this.updateSelectedFile}
+        onDoubleClick={() => this._openDiffView(file)}
+        style={{ ...style, color: 'var(--jp-warn-color0)' }}
```

Comment:
You should not pass style here. If needed pass a class name.

> The style are a necessity to handle list virtualization. But except for positioning, this is a bad practice to hard set style in code.

```suggestion
        style={{ ...style }}
```